ansible/roles/containerd/tasks/redhat.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: remove versionlock for containerd
-  command: yum versionlock delete containerd.io
+  command: yum versionlock {{ '--disablerepo "*" --enablerepo offline' if offline_mode_enabled else '' }} delete containerd.io
```

Comment:
yum versionlock commands try to download all mirrors from internet.  Adding logic for preventing yum from updating mirrors when using `yum versionlock` in air-gapped installations for "RHEL" ansible distributions.

Reply:
Wow so that means the rockylinux already comes wit Containerd installed? 🤔

Reply:
Discussed offline, the behavior for RockyLinux is different from CentoOS/RHEL `yum` and it tries to reach other repos when running `yum versionlock`.
But this is a good generic change that will prevent this bug in future RHEL releases too.